recipes-graphics/wayland/weston_4.0.0.imx.bb

```diff
@@ -84,6 +85,8 @@ PACKAGECONFIG[colord] = "--enable-colord,--disable-colord,colord"
 PACKAGECONFIG[clients] = "--enable-clients --enable-simple-clients --enable-demo-clients-install,--disable-clients --disable-simple-clients"
 # Weston with PAM support
 PACKAGECONFIG[pam] = "--with-pam,--without-pam,libpam"
+# Weston with i.MX G2D renderer
+PACKAGECONFIG[imxg2d] = "--enable-imxg2d,--disable-imxg2d,virtual/libg2d"
```

Comment:
Is this only working with imxgpu2d or any g2d provider?

Reply:
From what I understand, it works with any G2D provider.

Reply:
So shouldn't it be something like:
PACKAGECONFIG_append_virtual/libg2d = " imxg2d"
(the syntax is most likely wrong)

Reply:
Or just adding something like:
PACKAGECONFIG_append_imxdpu = " imxg2d"

Reply:
The syntax adds support for it but it does not /enable/ it.